docs/permissions/plugin-authors/05-frontend-authorization.md

```diff
@@ -149,11 +156,15 @@ Providing a disabled state can be a helpful signal to users, but there may be ca
             onChange={e => (title.current = e.target.value)}
           />
 -         {!loadingPermission && (
--           <Button disabled={!canAddTodo} variant="contained" onClick={handleAdd}>
+-           <Button
+-             disabled={!canAddTodo}
+-             variant="contained"
+-             onClick={() => onAdd(title.current)}
+-           >
 -             Add
 -           </Button>
 -         )}
-+         <Button variant="contained" onClick={handleAdd}>
++         <Button variant="contained" onClick={() => onAdd(title.current)}>
```

Comment:
This handler didn't exist. Other than this, the rest of the changes in this file are just formatting the new and old versions of each fragment